### ISSUE TITLE
Can anybody confirm that multi botting= ban?

### ISSUE THREAD

**Issue by icozmo** [finndev/PokeBuddy#898]
can anybody tell me if using multiple pokebuddy bots with a different location for each one leads to ip banning?

**Comment by tsebis** [comment]
yy

**Comment by Razdavitel** [comment]
Yes. If all the bots share the same IP they will get banned. This system is horrible because when you go out to a crowded city and play via 4G some people might share the same IP.

**Comment by icozmo** [comment]
is there a way to make each bot get different ip adress. through a vpn? I remember that "proxy cap" can do this If I may recall.

**Comment by Razdavitel** [comment]
Well you could use Proxy Cap for that. Never tried it myself but depending on how many bots you run perhaps you could set up a Virtual Machine for each bot.
Just take a windows OS that doesn't need a lot of requirements to run but is still supported by the bot.
